src/data/faq.json

```diff
@@ -542,20 +542,108 @@
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Will I lose my vaccination status if one of my certificates expires?",
+                                "anchor": "dcc_replacement_vac_status",
+                                "active": false,
+                                "textblock": [
+                                    "Your vaccination status does not change when a certificate expires. You only need to reissue the corresponding certificate in the app to be able to prove your vaccination status via the Corona-Warn-App in the future. To do this, follow the instructions in the overview of your certificates."
+                                ],
+                                "linkList": [
+                                    {
+                                        "title" : "Related FAQ articles:",
+                                        "links" : [
+                                            { "url" : "#dcc_replacement", "text": "Warum werden Zertifikate neu ausgestellt?"},
+                                            { "url" : "#dcc_replacement_howto", "text": "Wie und wann können Zertifikate in der App neu ausgestellt werden?"},
+                                            { "url" : "#dcc_replacement_which", "text": "Welche digitalen COVID Zertifikate der EU werden neu ausgestellt?"},
+                                            { "url" : "#dcc_replacement_issues", "text": "I cannot renew a certificate in the app. What could be the reason?"},
+                                            { "url" : "#glossary_validity", "text": "Gültigkeit eines EU digitalen COVID-Zertifikats."}
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "I cannot renew a certificate in the app. What could be the reason?",
+                                "anchor": "dcc_replacement_issues",
+                                "active": false,
+                                "textblock": [
+                                    "<ul> <li>Only certificates relevant to vaccination status can be renewed, such as booster vaccinations or convalescent certificates.</li> <li>The certificate is valid even longer than 28 days.</li> <li>The certificate has already expired more than 90 days ago. In this case, reissuing within the app is no longer possible.</li> <li>Only certificates issued in Germany can be renewed via the app.</li> <li>The number of new exhibitions is limited to three new exhibitions per user.</li> <li>Your Corona-Warn-App has not been updated to version 2.23. Please perform an update.</li> </ul>"

```

Comment:
"convalescent certificates" =>
"recovery certificates"

"convalescent certificates" is not an official name

Reply:
"The certificate has already expired more than 90 days ago."

remove the word "has" from the sentence.

Reply:
"The number of new exhibitions is limited to three new exhibitions per user."

"Exhibition" is completely the wrong word here!